### ISSUE TITLE
remove WAIT_FOR from 60-readahead.rules

### ISSUE THREAD

**Issue by sharkcz** [ibm-s390-tools/s390-tools#27]
The WAIT_FOR action has been removed from udev quite long time ago, but it's still used in https://github.com/ibm-s390-tools/s390-tools/blob/master/etc/udev/rules.d/60-readahead.rules#L10

It's causing errors like
`````15:24:33,372 ERR systemd-udevd:Invalid rule /usr/lib/udev/rules.d/60-readahead.rules:10: unknown key 'WAIT_FOR'`````
in system log (journal).

**Comment by xnox** [comment]
```
# on device add set initial readahead to 512 (instead of in kernel 128)
KERNEL=="sd*[!0-9]", WAIT_FOR="queue/read_ahead_kb", ATTR{queue/read_ahead_kb}="512"
```

Surely we can patch kernel to update the default to the new recommendation? Why kernel should not set this to 512, given that we force it to be that anyway?

**Comment by hoeppnerj** [comment]
Issue fixed via 13596f18da2194186423696d3c00592f591d0fa1
